src/events/SDL_pen.c

```diff
@@ -363,10 +370,54 @@ void SDL_SendPenTouch(Uint64 timestamp, SDL_PenID instance_id, const SDL_Window
             event.ptouch.down = down;
             SDL_PushEvent(&event);
         }
+
+        #if SYNTHESIZE_PEN_TO_MOUSE || SYNTHESIZE_PEN_TO_TOUCH
```

Comment:
Why are these ifdefs?

Reply:
They were in the touch code, too. They don't have to be there.

Reply:
Yeah, let's yank them.

Reply:
Gone in 3ec17e14f7c73747b31d2d26a82c16c0fe4212d0.